src/lib/types.h

```diff
@@ -792,4 +792,13 @@ typedef struct rnp_key_protection_params_t {
     pgp_hash_alg_t    hash_alg;
 } rnp_key_protection_params_t;
 
+/*
+ * Data structure for storing pointer to the buffer together with
+ * it's length.
+ */
+typedef struct buf_t {
+  uint8_t *bx;
```

Comment:
I don't really get the field name `bx`, I can't think of what that is meant to stand for. We have `buf` in `pgp_memory_t` that makes a bit more sense to me (or `ptr`, or something). What does it mean here?

Reply:
pointer to buffer. I prefer short names, I'll rename it to pbuf